docs/installation-guide-mac.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
+
+Installation guide for macOS
+============================
+
+Follow these instructions to install the CityEnergyAnalyst (CEA) on a macOS system (tested with Mojave 10.14.5) from the source
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Download and install Git (64-bit).

```

Comment:
actually, just typing `git` in the `terminal` app start a prompt "Would you like to install the tools now"?  for developer tools... that git should be enough!

Reply:
Can't say for sure, but that did not appear to work on my terminal until I manually installed. I might be wrong.